### DISCUSSION TITLE
Upgrading causing upgrade of dependencies is confusing.

### DISCUSSION THREAD

**Discussion by rdp** [Homebrew/discussions#2000] [Everyday usage]
Hello.

Got this today:

```
 % brew install maven
Updating Homebrew...
...
==> Homebrew was updated to version 3.2.7
==> Upgrading maven
  3.8.1 -> 3.8.2 

######################################################################## 100.0%
==> Downloading https://ghcr.io/v2/homebrew/core/openjdk/blobs/sha256:fb57725b9d1ac7dc846842c4905630ebe6878242cd876dc23d00d3100e6d4e26...
```

I'm guessing that jdk is a maven dependency.  It's a bit confusing that it would also update a dependency when I specified which package to upgrade.  Maybe homebrew could output some explanatory message before doing so like `Updating dependencies...`

Cheers!

**Comment by carlocab** [comment]
I believe it should show `Upgrading maven dependency openjdk...` or something like that when it's finished downloading things and is actually installing something.

**Comment by rdp** [comment]
It does show it later:
`==> Installing dependencies for maven: openjdk`
Which is OK...just takes 10 minutes to download it so I'm scratching my head initially that's all... :)
